src/spec.cr

```diff
@@ -78,6 +78,8 @@ require "./spec/dsl"
 module Spec
 end
 
+Spec.use_colors = Colorize.on_tty_only!
```

Comment:
In fact, I think `Spec.use_colors` can be removed, since `Colorize.enabled` exists. Just call `Colorize.on_tty_only!` once in `spec.cr` and the rest should sort itself out.

Reply:
done